hbase-server/src/main/java/org/apache/hadoop/hbase/backup/HFileArchiver.java

```diff
@@ -439,7 +439,7 @@ private static List<File> resolveAndArchive(FileSystem fs, Path baseArchiveDir,
 
     List<File> failures = new ArrayList<>();
     String startTime = Long.toString(start);
-    for (File file : toArchive) {
+    toArchive.parallelStream().forEach(file -> {
```

Comment:
Good to check the difference between the parallelStream() and stream().parallel(). Java doc say that for parallelStream "`It is allowable for this method to return a sequential stream.`"